src/clients/customer/lib/ui/home/home_viewmodel.dart

```diff
@@ -1,3 +1,30 @@
+import 'package:customer/api/firestore_api.dart';
+import 'package:customer/app/app.locator.dart';
+import 'package:customer/exceptions/firestore_api_exception.dart';
+import 'package:customer/models/application_models.dart';
 import 'package:stacked/stacked.dart';
+import 'package:customer/app/app.logger.dart';
 
-class HomeViewModel extends BaseViewModel {}
\ No newline at end of file
+class HomeViewModel extends FutureViewModel {
```

Comment:
Pass the type to return as a generic parameter to the `FutureViewModel<List<Merchant>>`